packages/api/models/user.js

```diff
@@ -1,20 +1,42 @@
 'use strict';
-
-const { Model } = require('sequelize');
+const { Model, Sequelize } = require('sequelize');
 module.exports = (sequelize, DataTypes) => {
-  class User extends Model {}
+  class User extends Model { }
 
   User.init(
     {
-      email: DataTypes.STRING,
-      username: DataTypes.STRING,
+      id: {
+        type: DataTypes.UUID,
+        defaultValue: Sequelize.UUIDV4,
+        allowNull: false,
+        primaryKey: true,
+      },
+      email: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        unique: {
+          args: true,
+          msg: 'Email already in use!',
+        },
+        validate: {
+          isEmail: true,
+          notEmpty: true,
+        },
+      },
+      username: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+        },
+      },
       password: DataTypes.STRING,
```

Comment:
I know you didn't write this schema, but as I think about this more deeply, I'm wondering what we're planning to use this endpoint for. What is the username? What is the password being used for? Aren't we only using Google oauth on the frontend? It seems like our user/authentication mechanism on the backend should be tied into that same oauth token scheme. If we do want to let current users add users, what would that look like?

These seem like valuable questions to answer when we're creating users.

Regardless, if we're going to introduce this endpoint as is, we shouldn't store passwords as plain text. You'll want to [encrypt](https://en.wikipedia.org/wiki/Bcrypt) this before storing it.